ng2-crypto-trading/src/app/crypto-table/crypto-table.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from "rxjs";
+
+import { CoincapService } from '../coincap.service';
+import { SortingService } from '../sorting.service';
+import { config } from '../app.config';
+
+import { CryptoData, Headers } from '../app.classes';
+
+@Component({
+  selector: 'app-crypto-table',
+  templateUrl: './crypto-table.component.html',
+  styleUrls: ['./crypto-table.component.css']
+})
+export class CryptoTableComponent implements OnInit {
+
+  data: CryptoData[] = [];
+  lastUpdate: Date;
+  activeColumn: string;
+  isDesc: boolean = false;
+  direction: number;
+  headers: Headers[] = config.headers;
+
+  constructor(private coincapService: CoincapService,
+              private sortingService: SortingService) { }
+
+  ngOnInit() {
+    this.getData();
+    this.update()
+      .subscribe();
+  }
+
+  getData() {
+    return this.coincapService.search()
+      .subscribe(coincapResp => {
+        let coincapArr = coincapResp.json();
+        this.data = [];
```

Comment:
dlaczego na początek przypisujesz tutaj pustą arraykę?

Reply:
właściwie zapomniałam usunąć, bo wcześniej robiłam push do this.data, teraz przypisuję, więc wyrzucę